src/SimpleNameRegister.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+///@title An on-chain name registry
+///@author Calnix
+contract SimpleNameRegister {
+    
+    /// @notice map a name to an address to identify current holder 
+    mapping (string => address) public holder;    
+
+    /// @notice emit an event when a name is registered or released
```

Comment:
usually natspec is for the thing right below it. in this case you should put that notice above each event.

if you did want to make a more general comment about both of them (which I don't recommend in this situation) then you can make a non-natspec comment.

Natspec comments are either full line comments with triple /
```
/// @notice This is a natspec comment.
```
or multi-line comments starting with /** 
```
/** 
@notice This is a natspec comment.
 */
 
 A non-natspec comment would use `//` for a full line comment and `/* */` for multi line.